modules/core/client/views/lightbox/contact.client.view.html

```diff
@@ -24,6 +24,17 @@
 
         </div>
 
+        <div class="row">
+
+              <div class="input-field col s12">
+
+                  <textarea id="textarea1" class="materialize-textarea"></textarea>
+                  <label for="textarea1">What would you like know?</label>
```

Comment:
@dnol777 
ahhh no I wasn't saying to add quotes - there should be a 'to' in between 'like' and 'know'...